src/utils.js

```diff
@@ -0,0 +1,3 @@
+export const extend = (a, b) => {
+  return Object.assign({}, a, b);
+};
```

Comment:
почему бы не сократить? :)
```suggestion
export const extend = (a, b) => ({a, b});
```

Reply:
Попробовал сокращенную запись - получился некорректно схлопнутый обьект:
Object {
 "a": Object { "maxMistakes": 3,  "mistakes": 0, },
 "b": Object { "mistakes": 0, },  
}.
Норм обьект получается только через spread: (a, b) => ({...a, ...b}); 
но мне кажется assign более нативный/производительный

Reply:
Точно, там же обьекты... spread читается в коде, но это - сугубо дело вкуса 👍🏻